ChangeLog.md

```diff
@@ -238,6 +238,9 @@ The main focus of this release was revamping the build system and bootstrapping.
 * Happy is about 5 times faster on large examples, due to some
   changes in the LALR(1) algorithms.
 
+As of version 1.5, Happy is capable of parsing full Haskell.
+We have a Haskell parser that uses Happy, which will shortly be part of the library collection distributed with GHC.
```

Comment:
Oh I was moving text from the old website. This is something presumably @simonmar wrote years ago — "shortly" probably meant something 10-20 years ago! I just wanted to document in posterity that this was the intent at that point, for archival purposes, and also because I personally think it was a good idea that still ought to happen.